eclipse/src/saros/editor/EditorManager.java

```diff
@@ -1062,6 +1054,8 @@ private void replaceText(SPath path, int offset, String replacedText, String tex
 
       // Try to replace
       try {
+        // Attention: This method also alters the annotation model if an
+        // annotation exists at the offset and the length &gt; 1
```

Comment:
Use correct symbol instead of character code?
```suggestion
        // annotation exists at the offset and the length > 1
```